apps/kg/models/mxnet/score_fun.py

```diff
@@ -5,16 +5,20 @@
 from mxnet import ndarray as nd
 
 class TransEScore(nn.Block):
-    def __init__(self, gamma):
+    def __init__(self, gamma, dist_func='l2'):
```

Comment:
can you make the same modification for MXNet?

Reply:
You means supporting l2 distance for transE? The PR already has it.

Reply:
i mean, can we have the similar implementation as the one in Pytorch version? so mxnet version and pytorch version has similar speed.

Reply:
> i mean, can we have the similar implementation as the one in Pytorch version? so mxnet version and pytorch version has similar speed.

Done.